src/pages/index.js

```diff
@@ -9,7 +9,8 @@ export default function HomePage() {
 				<title key="title">My Project</title>
 				<meta key="description" name="description" content="This is my project" />
 			</Head>
```

Comment:
I would recommend to build your homepage in this order
```
<Head> 
     <title>
     <meta> 
</Head>
<Layout>
     <h1>
</Layout>
```

But nevertheless, usually you would include your Layout an Head in you _app.js instead of your index.js
The index.js is the root file of your application where you code most of the logic, where you should manage your states etc.
This means the code you wrote in your form component actually belongs here.